### DISCUSSION TITLE
Magic links / one time autorisation

### DISCUSSION THREAD

**Discussion by legaljustus** [Budibase/budibase#1600] [Ideas]
I was wondering if there is a way to implement magic links for user logins like [this](https://magic.link/posts/hello-world) or other custom user auth.

It looks like it should be pretty simple to integrate and would be quite a useful feature.

Looking forward to what you think!

**Comment by joebudi** [comment]
Hey @legaljustus . We use http://www.passportjs.org/ for our authentication system. Could, we only offer Google/basic Auth. We will hope to create further Auth options in the future. Magic links are not on the roadmap. You could potentially implement them yourself rather than wait for us:
https://reposhub.com/javascript/security/mxstbr-passport-magic-login.html

I've not read the above post but seems to be on topic.
